Service/Request/RequestFactory.php

```diff
@@ -123,18 +123,21 @@ private function getRequestUri(ServiceRequestInterface $serviceRequest)
         $endpoint = $this->endpointRegistry->getEndpoint($serviceRequest);
         $baseUrl = $endpoint->getBaseUrl();
         $path = $endpoint->getPath();
+        $queryParams = $this->parseQueryParams($path);
 
         //check for placeholders
         preg_match_all('/{(\w*)}/', $path, $matches);
         foreach ($matches[0] as $index => $placeholder) {
-            $getterMethod = 'get'.ucfirst($matches[1][$index]);
+            $key = $matches[1][$index];
```

Comment:
isset check needed

Reply:
Doesn't look so. @vladimir-chmykov let's roll this line back, so people don't discuss not related changes.

Reply:
Not really, if there's a `$index` in `$matches[0]` it's gonna be in `$matches[1]` as well.

Reply:
agreed, it's a built-in function with certain output